### PR TITLE
add tool to generate emoji name data

### PR DESCRIPTION
**Pull request by dougfelt**
.This is to support the website, we want to generate a table of names for those emoji images that we want to display.

### CODE REVIEW

File: nototools/generate_emoji_name_data.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+
+def unicode_name(cp):
+  name = unicode_data.name(cp).title()
+  name = name.replace('Oclock', "O'Clock")
+  name = name.replace('Mans', "Man's")
+  name = name.replace('Womans', "Woman's")
+  name = name.replace('Mens', "Men's")
+  name = name.replace('Womens', "Women's")
+  return name
```
Comment (roozbehp) [inline nototools/generate_emoji_name_data.py]: I would change the data to a dictionary and loop over it.

File: nototools/generate_emoji_name_data.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    if len(cps) == 1:
+      name = unicode_name(cps[0])
+    elif is_flag_sequence(cps):
+      name = flag_sequence_name(cps)
+    else:
+      name = ''
+    output[tuple(cps)] = (fname, sequence, name)
+
+  with open(outfile, 'w') as f:
+    f.write('[\n')
```
Comment (jamesgk) [inline nototools/generate_emoji_name_data.py]: Any reason we're not using the Python json library here?

Comment (dougfelt) [inline nototools/generate_emoji_name_data.py]: So I don't have to build objects from the data just so the json library can decode them to generate the output?  The data is really simple so it didn't seem necessary.

File: nototools/generate_emoji_name_data.py (hunk 1 above)
Comment (dougfelt) [inline nototools/generate_emoji_name_data.py]: OK, changed that.

File: nototools/generate_emoji_name_data.py (hunk 2 above)
Comment (jamesgk) [inline nototools/generate_emoji_name_data.py]: Yea I guess it's simple either way. I would always tend towards using a library for peace of mind. But if we don't use it here we have an unused `import json` statement above.

**Comment by jamesgk** [comment]
LGTM

**Pull request merged**